UITests/TransferMethod/AddTransferMethodBankCardTests.swift

```diff
@@ -14,52 +14,59 @@ class AddTransferMethodBankCardTests: BaseTests {
         mockServer.tearDown()
     }
 
-    private func setUpAddTransferMethodBankCardScreen() {
-        selectTransferMethodType = SelectTransferMethodType(app: app)
-        addTransferMethod = AddTransferMethod(app: app, for: .debitCard)
+    func testAddTransferMethod_createBankCard() {
+        mockServer.setupStub(url: "/rest/v3/users/usr-token/bank-cards",
+                             filename: "BankCardResponse",
+                             method: HTTPMethod.post)
 
-        app.tables.cells.containing(.staticText, identifier: "Add Transfer Method").element(boundBy: 0).tap()
-        spinner = app.activityIndicators["activityIndicator"]
-        waitForNonExistence(spinner)
-        selectTransferMethodType.selectCountry(country: "United States")
-        selectTransferMethodType.selectCurrency(currency: "US Dollar")
+        addTransferMethod.setCardNumber("4895142232120006")
+        addTransferMethod.setDateOfExpiry(expiryMonth: "March", expiryYear: "2020")
+
+        XCTAssertEqual(app.textFields["dateOfExpiry"].value as? String, "03/20")
+
+        addTransferMethod.setCvv("022")
+        addTransferMethod.clickCreateTransferMethodButton()
 
-        app.tables["transferMethodTableView"].staticTexts.element(matching: debitCard).tap()
         waitForNonExistence(spinner)
+
+        XCTAssert(app.navigationBars.staticTexts["Account Settings"].exists)
     }
 
-    func testAddTransferMethod_createDebitCard() {
+    func testAddTransferMethod_createBankCardBusiness() {
         mockServer.setupStub(url: "/rest/v3/users/usr-token/bank-cards",
```

Comment:
Same thing here.

Reply:
I am sorry, but I am not able to catch the context.
Same thing for what?